torch/csrc/jit/script/exit_transforms.cpp

```diff
@@ -254,45 +332,73 @@ struct ExitTransformer {
     destroyNodeAfterExit(*iter);
   }
 
+  // if we are transforming LoopContinuations, then when we enter a loop
+  // the exits target that loop. if transforming ReturnStmts, exit target
+  // the most recent Closure/Graph
+  void setTargetBlock(Block* block) {
+    if (current_exit_kind_ == prim::LoopContinuation) {
+      if (owningNodeKind(block) == prim::Loop) {
+        target_block_ = block;
+      }
+    } else if (current_exit_kind_ == prim::ReturnStmt) {
```

Comment:
It is possible to have `current_exit_kind_ == prim::LoopContinuation` but `owningNodeKind(block) != prim::Loop `, right? In this case we're just not updating the `target_block_`? Is my understanding correct? If yes, could you please add this to the comment and also maybe rename this function to `update` instead of `set`.